src/components/common/user-button.tsx

```
@@ -0,0 +1,51 @@
+type Session = Awaited<ReturnType<typeof auth.api.getSession>>
+
+const UserButton = ({ session }: { session: Session }) => {
+    const router = useRouter()
+    const initials = useMemo(() => {
+        if(!session?.user.name) return ''
+
+        return session.user.name.split(' ').map(name => name[0]).join('')
+    }, [session?.user.name])
+
+    return (
+        <div role="navigation" aria-label="User menu">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<nav>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)